mdk/src/main/java/com/example/examplemod/ExampleMod.java

```diff
@@ -33,7 +35,7 @@ public ExampleMod() {
         // Register the processIMC method for modloading
         FMLJavaModLoadingContext.get().getModEventBus().addListener(this::processIMC);
         // Register the doClientStuff method for modloading
-        FMLJavaModLoadingContext.get().getModEventBus().addListener(this::doClientStuff);
+        // FMLJavaModLoadingContext.get().getModEventBus().addListener(this::doClientStuff);
```

Comment:
A cleaner solution might just be to wrap this line inside of
```java
DistExecutor.runWhenOn(Dist.CLIENT, () -> () -> {
    FMLJavaModLoadingContext.get().getModEventBus().addListener(this::doClientStuff);
});
```

Though other people should potentially comment on this as well.

Reply:
Oh, it look good as general solve. But is annotation good in this case?